public/app/plugins/datasource/graphite/query_ctrl.ts

```diff
@@ -437,6 +439,14 @@ export class GraphiteQueryCtrl extends QueryCtrl {
       dispatch(notifyApp(createErrorNotification(`Fetching tags failed: ${error.message}.`)));
     }
   }
+
+  private handleMetricsAutoCompleteError(error: Error): void {
+    console.log(error);
```

Comment:
```suggestion
    console.error(error);
```

Reply:
I have noticed that we are using `console.log` in `private handleTagsAutoCompleteError`. I think it would make sense to change it to `console.error` as well.

Reply:
Coolio, I'll do the same for `handleTagsAutoCompleteError` :)